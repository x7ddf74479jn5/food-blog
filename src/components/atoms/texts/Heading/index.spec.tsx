import renderer from "react-test-renderer";

import { HeadingOne } from ".";

describe("components/atoms/texts/HeadingOne", () => {
  const text = "Heading 1";
  it("snapshot", () => {
    const tree = renderer.create(<HeadingOne>{text}</HeadingOne>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
