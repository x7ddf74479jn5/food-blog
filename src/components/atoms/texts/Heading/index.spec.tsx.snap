// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/texts/HeadingOne snapshot 1`] = `
<h1
  className="flex justify-center text-4xl font-bold"
>
  Heading 1
</h1>
`;
